### PR TITLE
Implement computed attributes

### PR DESCRIPTION
**Pull request by Abyss777**
Unfortunately its name intersects with just attributes. Named in the following way:
`[object]Attributes` named shortly, in other places they are `ComputedAttributes`

Chose http://fontawesome.io/icon/tasks/ glyph

image

image

Dialog now contain only text filelds
image

I'm going to implement "Known Position Attributes" store to use combobox for `attribute` field and automatically select appropriate `type` in this dialog.

### CODE REVIEW

File: web/app/view/edit/UsersController.js [hunk 1]
```diff
@@ -155,13 +156,31 @@ Ext.define('Traccar.view.edit.UsersController', {
         }).show();
     },
 
+    onAttributesClick: function () {
+        var user = this.getView().getSelectionModel().getSelection()[0];
+        Ext.create('Traccar.view.BaseWindow', {
+            title: Strings.sharedCalendars,
```
Comment (tananaev) [inline web/app/view/edit/UsersController.js]: This seems to be a wrong string key.

File: web/app/view/permissions/DeviceAttributes.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+Ext.define('Traccar.view.permissions.DeviceAttributes', {
+    extend: 'Traccar.view.permissions.Base',
+    xtype: 'deviceAttributesView',
+
+    columns: {
+        items: [{
+            text: Strings.sharedDescription,
```
Comment (tananaev) [inline web/app/view/permissions/DeviceAttributes.js]: Maybe we should have an attribute key in the table as well?

Comment (tananaev) [inline web/app/view/permissions/DeviceAttributes.js]: Same applies to the other similar tables.

**Comment by Abyss777** [comment]
Sorry for missed string
Added `attribute` column, looks fine

image

Agree about huge amount of duplicate. We can gradually deduplicate it. I'll think over what can be done.

**Comment by tananaev** [comment]
Merged, thanks. I agree that we can always remove duplication and improve things later.

**Pull request merged**